components/StyledTextarea.js

```diff
@@ -9,6 +9,9 @@ import Container from './Container';
 import StyledTag from './StyledTag';
 
 const TextArea = styled.textarea`
+  font-family: 'inherit';
```

Comment:
This is making my `TextArea` render with a serif font.

Reply:
Interesting, I added that to avoid the special font used by some browsers for textareas. Anyway it seems it was a dev-environment only issue and https://github.com/opencollective/opencollective-frontend/pull/3564 fixed it. I'll remove this line.